cmake/Modules/GrCUDA.cmake

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Free Software Foundation, Inc.
```

Comment:
We can drop the FSF for copyright, and you can put Author/BlackLynx

Reply:
Okay. I think that came from copy-paste.  By the way, we did complete "registration" with the FSF for the purposes of contributing to GR.